app/app.go

```diff
@@ -736,12 +736,16 @@ func (*App) BlockedParams() [][2]string {
 	return [][2]string{
 		// bank.SendEnabled
 		{banktypes.ModuleName, string(banktypes.KeySendEnabled)},
-		// staking.UnbondingTime
-		{stakingtypes.ModuleName, string(stakingtypes.KeyUnbondingTime)},
-		// staking.BondDenom
-		{stakingtypes.ModuleName, string(stakingtypes.KeyBondDenom)},
 		// consensus.validator.PubKeyTypes
 		{baseapp.Paramspace, string(baseapp.ParamStoreKeyValidatorParams)},
+		// consensus.Version.AppVersion
+		{baseapp.Paramspace, string(baseapp.ParamStoreKeyVersionParams)},
+		// staking.BondDenom
+		{stakingtypes.ModuleName, string(stakingtypes.KeyBondDenom)},
+		// staking.MaxValidators
+		{stakingtypes.ModuleName, string(stakingtypes.KeyMaxValidators)},
+		// staking.UnbondingTime
+		{stakingtypes.ModuleName, string(stakingtypes.KeyUnbondingTime)},
```

Comment:
[blocking] this is a breaking change so if it is included in this PR then the PR title needs a `!` to conform to conventional commits.

IMO we shouldn't add this to the blocked list yet. We should create a GH issue with the list of params that are governance modifiable even though the specs claim they shouldn't be. We should get team alignment on what to do for those params. Then we may update this list or update the specs accordingly. cc: @evan-forbes

Reply:
yeah any consens breaking change needs a CIP

@fahimahmedx, this is consensus breaking since nodes running this won't change a param, where nodes that are not will change the param

Reply:
the CIP doesn't have to be super in depth, since the change is pretty small, but we do still need to explain the benefits of adding or removing any params